pennylane/ops/op_math/controlled_ops.py

```diff
@@ -1082,7 +1082,7 @@ def _check_and_convert_control_values(control_values, control_wires):
 
 
 class MultiControlledX(ControlledOp):
-    r"""MultiControlledX(control_wires, wires, control_values, work_wires)
+    r"""MultiControlledX(wires)
```

Comment:
Why are `control_values` and `work_wires` missing here? 🤔

Reply:
Let me try removing this enforced docstr to allow sphinx default version

Reply:
@isaacdevlugt The default view looks a little bit clumsy though. This is because I set up an default value for the `wires` arg. What do you think? If we need to set it as positional one it might need some src code modification but should be simple doable very fast in this PR as well
![image](https://github.com/user-attachments/assets/3fab46f0-51bf-47f8-915a-45d72c9f2220)